egs2/TEMPLATE/asr1/pyscripts/audio/format_wav_scp.py

```diff
@@ -77,7 +77,7 @@ def utt2ref_channels(x, d=utt2ref_channels_dict) -> Tuple[int, ...]:
     if args.segments is not None:
         # Note: kaldiio supports only wav-pcm-int16le file.
         loader = kaldiio.load_scp(args.scp, segments=args.segments)
-        with SoundScpWriter(args.outdir, args.name,
+        with SoundScpWriter(args.outdir, Path(args.outdir) / f'{args.name}.scp',
```

Comment:
There is still a minor bug for the segment version to take effect.  I fixed here, please have a check @kamo-naoyuki Thank you!

Reply:
Thank you! Looks good.
I forgot that I changed the interface of SoundScpWriter.